app/controllers/state_file/landing_page_controller.rb

```diff
@@ -18,7 +18,10 @@ def edit
     end
 
     def update
-      sign_out current_intake if current_intake.present?
+      all_current_intakes = StateFile::StateInformationService.active_state_codes.map { |c| send("current_state_file_#{c}_intake".to_sym) }.compact
+      if all_current_intakes.present?
```

Comment:
non-blocking & minor, but i don't think you need this `.present?` check, since it'll be an array no matter what, and if there are no elements, the `.each` block will just not execute any times

Reply:
good point